File: 01-FrontEnd/PXBank-UI/src/app/shared/model/query-models/query-results.model.ts
export class QueryResultsModel {
	// fields
	items: any[];
	totalCount: number;
	errorMessage: string = "";
	obj: any;

	constructor(_items: any[] = [], _errorMessage: string = '', _obj: any = {}) {
		this.items = _items;
		this.totalCount = _items.length;
		this.obj = _obj;
	}
}

File: 02-BackEnd/PXBank.WebAPI/_Core/Models/QueryResultsModel.cs
namespace PXBank.WebAPI._Core.Models
{
    [Serializable]
    public class QueryResultsModel
    {
        public object items { get; set; }
        public int totalCount { get; set; }
        public string errorMessage { get; set; }
        public object obj { get; set; }
    }
}
